mdx/api/index.mdx

```diff
@@ -889,24 +830,17 @@ Retrieves a list of orders given query parameters.
-| `makerFeeAssetData`     | (Optional) Identifying token information of the asset(s) the maker must pay to the feeRecipientAddress.                                                                                                                                                                                                                                           |
-| `takerFeeAssetData`     | (Optional) Identifying token information of the asset(s) the taker must pay to the feeRecipientAddress.                                                                                                                                                                                                                                           |
-| `senderAddress`         | (Optional) An advanced field that doesn't need to be set. It allows the maker to enforce that the order flow through some additional logic before it can be filled (e.g., a KYC whitelist) -- more on the ability to extend 0x later.                                                                                                             |
-
+| Field                   | Description                                                                                                                                                                                                                                                                                                                                        |
+| ----------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `makerToken`            | (Optional) The address of the ERC20 token the maker is selling and the maker is selling to the taker. Example `0xe41d2489571d322189246dafa5ebde1f4699f498`
+| `takerToken`            | (Optional) The address of the ERC20 token the taker is selling and the taker is selling to the maker. Example `0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2`
+| `makerAmount`           | (Optional) The amount of makerToken being sold by the maker.
+| `takerAmount`           | (Optional) The amount of takerToken being sold by the taker.
+| `maker`                 | (Optional) The address of the maker, and signer
+| `taker`                 | (Optional) Allowed taker address
+| `pool`                  | (Optional) The staking pool to attribute the 0x protocol fee from this order. Set to zero to attribute to the default pool, not owned by anyone. TODO FIX THIS
+| `feeRecipient`          | (Optional) Recipient of maker token or taker token fees (if non-zero)
+| `trader`                | (Optional) The address of either the maker or the taker
```

Comment:
I think we could get rid of the fields here and refer to SignedOrder instead?
Or if we keep this, list only a subset that are queryable. e.g. `makerAmount` and `takerAmount` return orders that match the exact amount, so there's not a sensible use case for them.

We could say:
```
Any [SignedOrder]() field can be use as a query parameter. Additional query parameters are listed below.

| Field  | Description       |
| ------ | ------ |
| `trader`   | (Optional) The address of either the maker or the taker |
```

Reply:
Great suggestion! I wasn't sure what to do with this table